pubs/pretty.py

```diff
@@ -64,8 +65,12 @@ def paper_oneliner(p, citekey_only=False):
         return p.citekey
     else:
         bibdesc = bib_oneliner(p.bibdata)
+        doc_str = ''
+        if p.docpath is not None:
+            doc_str = color.dye_out(' [{}]'.format(os.path.splitext(p.docpath)[1][1:]),
```

Comment:
Currently it is possible to add a document without an extension. I am not sure whether we should allow it in the first place or not but this would raise an exception here because of the `[1:]` indexing. You could just replace it with `'NOEXT'` or something here when the extension is empty.